File: Zadania z PDF'ów/Szyfr Cezara/Python/Podstawowy z uzyciem mod.py
import string

jawny = ""
szyfrogram = ""
klucz = 0

jawny = str(input("Podaj tekst do zaszyfrowania: "))
klucz = int(input("Podaj klucz: "))

for i in range(0, len(jawny)):
    if jawny[i] >= 'A' and jawny[i] <= 'Z':
        szyfrogram += chr((ord(jawny[i]) - ord('A') + klucz) % 26 + ord('A'))
    elif jawny[i] >= 'a' and jawny[i] <= 'z':
        szyfrogram += chr((ord(jawny[i]) - ord('a') + klucz) % 26 + ord('a'))

print(szyfrogram)

File: Zadania z PDF'ów/Szyfr Cezara/c++/Podstawowy z uzyciem mod.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string jawny, szyfrogram = "";
    int klucz;
    cout << "Podaj tekst do zaszyfrowania: ";
    cin >> jawny;
    cout << "Podaj klucz: ";
    cin >> klucz;
    for (int i = 0; i < jawny.length(); i++) {
        if (jawny[i] >= 'A' && jawny[i] <= 'Z') {
            szyfrogram += (jawny[i] - 'A' + klucz) % 26 + 'A';
        } else if (jawny[i] >= 'a' && jawny[i] <= 'z') {
            szyfrogram += (jawny[i] - 'a' + klucz) % 26 + 'a';
        }
    }
    cout << "Szyfrogram: " << szyfrogram << endl;
    return 0;
}
